specification/cognitiveservices/AnomalyDetector/multivariate/models.tsp

```diff
@@ -129,11 +129,9 @@ to be detected.
   dataSource: url;
 
   @doc("""
-Number of top contributed
-variables for one anomalous time stamp in the response. The default is
-10.
+Number of top contributed variables for one anomalous time stamp in the response.
 """)
-  topContributorCount: int32;
+  topContributorCount?: int32 = 10;
```

Comment:
@lmazuel is this actually correct? Seems this makes it optional with a default, but in pure C# parlance that's not correct. Wondering if TypeSpec is different. In C# this means the parameter is optional and will default to 10, but because the `?` it'd now allow null. What does the addition of both `?` and a default value mean to TypeSpec?

Reply:
The ? in TypeSpec makes the property optional. In TypeSpec, you could not give a default value for a property that is not optional.